### PR TITLE
Language query

### PR DESCRIPTION
**Pull request by harrisonpim**
- adds a dis_max block to the query, handling different language analysers
- adds some language-specific examples
- little bit of tidying

### CODE REVIEW

File: docs/arabic_examples.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- الكشف  
+  https://github.com/wellcomecollection/wellcome-collection-tei/blob/master/Arabic/WMS_Arabic_256.xml  
+  should be able to find with لكشف
+- معرفت  
+  https://github.com/wellcomecollection/wellcome-collection-tei/commit/4a4d6bcee91dae51bf7070e1bc20430e5c5b4f76  
+  should be able to find with عرف
```
Comment (jamesgorrie) [inline docs/arabic_examples.md:6]: Could put this into a table?

File: pages/api/search/works.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
       explain: true,
       source: {
         ...searchQuery.template.source,
+        size: 1000,
```
Comment (jamesgorrie) [inline pages/api/search/works.ts:46]: We sure?

File: queries/languages.ts [hunk 3]
```diff
@@ -23,29 +28,77 @@ const query = {
...
+                fuzziness: 'AUTO',
+                operator: 'And',
+                query: '{{query}}',
+                type: 'best_fields'
+              }
+            },
+            {
+              dis_max: {
+                _name: 'non-english titles',
+                queries: languageQueries
```
Comment (jamesgorrie) [inline queries/languages.ts:93]: Did we try use the multi-match query here? As far as I can tell it uses the same analyser that the data was indexed with. 

I think the [`best_filelds`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#type-best-fields) should deal with this, and make the query more succinct.

**Review by jamesgorrie — COMMENTED** [review]
Question on how we are constructing the language queries.

File: pages/api/search/works.ts (hunk 2 above)
Comment (harrisonpim) [reply pages/api/search/works.ts:46]: 🤦

File: queries/languages.ts (hunk 3 above)
Comment (harrisonpim) [reply queries/languages.ts:93]: looks like the results are the same for all of the tests, though the scores are very slightly different. haven't figured out exactly where the difference is coming from, but i'm fine with using that implementation if you're happier with it

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**